example/java/TransactionExample.java

```diff
@@ -0,0 +1,120 @@
+import iroha.protocol.Commands;
+import iroha.protocol.BlockOuterClass;
+import iroha.protocol.Endpoint;
+import iroha.protocol.Queries;
+
+import iroha.protocol.QueryServiceGrpc;
+import iroha.protocol.CommandServiceGrpc;
+import iroha.protocol.CommandServiceGrpc.CommandServiceBlockingStub;
+import iroha.protocol.Endpoint.TxStatus;
+import iroha.protocol.Endpoint.TxStatusRequest;
+import iroha.protocol.Endpoint.ToriiResponse;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import com.google.protobuf.ByteString;
+
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.lang.Thread;
+
+class TransactionExample {
+    static {
+        try {
+            System.loadLibrary("irohajava");
+        } catch (UnsatisfiedLinkError e) {
+            System.err.println("Native code library failed to load. \n" + e);
+            System.exit(1);
+        }
+    }
+
+    private static ModelCrypto crypto = new ModelCrypto();
+    private static ModelTransactionBuilder txBuilder = new ModelTransactionBuilder();
+    private static ModelQueryBuilder queryBuilder = new ModelQueryBuilder();
+    private static ModelProtoTransaction protoTxHelper = new ModelProtoTransaction();
+    private static ModelProtoQuery protoQueryHelper = new ModelProtoQuery();
+
+
+
+    public static void main(String[] args) {
+        String adminPub = "", adminPriv = "";
+
+        try {
+            adminPub = new String(Files.readAllBytes(Paths.get("../admin@test.pub")));
+            adminPriv = new String(Files.readAllBytes(Paths.get("../admin@test.priv")));
+        } catch (IOException e) {
+            System.err.println("Unable to read key files.\n " + e);
+            System.exit(1);
+        }
```

Comment:
can this code be added to `irohajava` package?
so the user writes something like `irohajava.load()`

Reply:
User loads library via ```System.loadLibrary("irohajava");``` but how can library be aware of user's keys and their location?